source/installation/install-on-a-single-machine/install-ubuntu.txt

```diff
@@ -1,5 +1,76 @@
+.. _single-ubuntu:
+
 ==============
 Install Ubuntu
```

Comment:
```suggestion
Install on Ubuntu
```

Reply:
I think we should do this for all four task pages, because we're not installing the OS, but instead installing "on" the OS.

Reply:
Done.